package.json

```diff
@@ -155,6 +155,7 @@
     "user-detail": "/users/detail/:username",
     "my-user-access": "/my-user-access",
     "access-requests": "/access-requests",
-    "access-requests-detail": "/access-requests/:requestId"
+    "access-requests-detail": "/access-requests/:requestId",
+    "resources": "/resources"
```

Comment:
Not sure how to make this show up in the side nav

Reply:
That has to be set up in cloud services config repo: https://github.com/RedHatInsights/cloud-services-config

Although for demo purposes, we can host a local version of it or just access the page via URL directly.